oci/private/pull.bzl

```diff
@@ -34,6 +34,11 @@ _IMAGE_REFERENCE_ATTRS = {
         default = "@oci_auth_config//:standard_authorization_config_path",
         allow_single_file = True,
     ),
+    "docker.elastic.co": {
```

Comment:
this is the wrong place to put it. it should go into _WWW_AUTH

Reply:
https://github.com/bazel-contrib/rules_oci/blob/f191ce2b08556e1d581db3cb92f7207b41aa40b0/oci/private/pull.bzl#L76

Reply:
face_palm